src/typescript/sdk/src/indexer-v2/queries/app/home.ts

```diff
@@ -48,6 +63,25 @@ const selectMarketStates = ({
   return query;
 };
 
+selectMarketHelper({
+  tableName: TableName.PriceFeed,
+});
```

Comment:
Wait, why is this function randomly called here ?

Reply:
Removed, thx